Formula/mist.rb

```diff
@@ -0,0 +1,31 @@
+class Mist < Formula
+  desc "Mac command-line tool that automatically downloads macOS Installers / Firmwares"
+  homepage "https://github.com/ninxsoft/Mist"
+  url "https://github.com/ninxsoft/Mist/archive/refs/tags/v1.6.1.tar.gz"
+  sha256 "0b1ec7fed7bbafb3221656376bf69407fe602ad4cdb938089d8cf7c585112394"
+  license "MIT"
+
+  head "https://github.com/ninxsoft/Mist.git", branch: "main"
+
+  # Mist requires Swift 5.5
+  depends_on xcode: ["13.1", :build]
```

Comment:
```suggestion
  depends_on xcode: ["13.1", :build]
  uses_from_macos "swift"
```
Might be enough to get this to work on Linux. Maybe.

Reply:
I guess one thing I'll mention is that Mist's functionality on Linux would be somewhat limited - it relies on functionality of macOS itself (its installer framework) in order to be able to assemble macOS Installer apps.

Would it make more sense to specify that this formula (at least for now) is only supported on macOS brew and not Linux brew?